stable/aws-load-balancer-controller/templates/deployment.yaml

```diff
@@ -60,15 +99,22 @@ spec:
           readOnly: true
         ports:
         - name: webhook-server
-          containerPort: 9443
+          containerPort: {{ .Values.webhookBindPort | default 9443 }}
           protocol: TCP
         - name: metrics-server
-          containerPort: 8080
+          containerPort: {{ (split ":" .Values.metricsBindAddr)._1 | default 8080 }}
+          protocol: TCP
+        {{- if .Values.livenessProbe }}
+        - name: health
```

Comment:
any reason this port needs to be exposed?

Reply:
The reason is that I wanted to use this name in the livenessProbe port.

When the healthcheck port is can be changed by this PR(`healthProbeBindAddr`), the livenessProbe port also needs to be changed, but I wanted to give it a name so that no changes are missed.

https://github.com/aws/eks-charts/pull/312/files#diff-16853b1bc086bb28c85a18bece05525988df3c4d6caeb4d12e3596979fa3a996R129

Reply:
I understand. I'll just fix the part about `health-probe-bind-addr` for now.

Reply:
It isn't arg(health-probe-bind-addr) I wanted to use right away, so I deleted the arg and rebased.
ab79529
f709626